docs/ami.md

```diff
@@ -1,6 +1,6 @@
 # Amazon AMIs
 
-AMIs for RancherOS are published under the owner ID `275947076441` in the `us-west-1` and `us-west-2` regions
+AMIs for Elemental are published under the owner ID `275947076441` in the `us-west-1` and `us-west-2` regions
```

Comment:
I think the Ami stuff can be fully dropped as we are not publishing anything on aws anymore and the owner I'd gas changed for sure

Reply:
Nice, will drop the whole thing!